### ISSUE TITLE
Порядок колонок в таблице с проводками

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#2348]
Originally reported on Google Code with ID 2348

```

Порядок колонок в таблице с проводками:

http://gsbelarus.com/gs/modules.php?name=Forums&file=viewtopic&p=2517#2517

```

Reported by `gs1994` on 2011-02-28 11:47:52

**Comment by gsbelarus** [comment]
```
У нас есть две ф-ции invMat_LoadFrmSettings и invMat_SaveFrmSettings, соответственно
их надо добавить в Tgdc_frmInvDocument нужного подтипа в LoadSettingsAfterCreate и
SaveSettings. 

Конечно по хорошему нужно было создать данный ф-ционал в базовой форме складского документа
в exe.
```

Reported by `Alexander.GoldenSoft` on 2011-03-02 06:58:51

**Comment by gsbelarus** [comment]
```

  Было бы очень удобно !
  Добавлю, что "создать данный ф-ционал в базовой форме складского документа в exe",
а бывают же и не складские документы, по которым формируются бухгалтерские проводки.
Так что необходимо предусмотреть такую возможность в любом документе. На выбор разработчика.

```

Reported by `NikolayUkleyko` on 2013-05-17 12:16:16

**Comment by gsbelarus** [comment]
Reported by `NikolayUkleyko` on 2013-11-14 12:09:02
- **Status changed**: `Duplicate`
- **Merged into**: #2462
